### ISSUE TITLE
Add support for more CPU architectures

### ISSUE THREAD

**Issue by eran101** [AlmaLinux/docker-images#4]
Please add support for more CPU architectures.
Especially ARM64 which will allow it to run on the newer Raspberry Pi.

**Comment by ezamriy** [comment]
Unfortunately, AlmaLinux doesn't support ARM architecture yet so this issue is blocked by upstream.

We are expecting it somewhere in Q2.

**Comment by ezamriy** [comment]
The aarch64 architecture support is added to both Docker Hub and Quay.io images:

* https://hub.docker.com/_/almalinux
* https://quay.io/repository/almalinux/almalinux?tag=latest&tab=tags

We are going to work on the ppc64le support in Q3, but let's create another task when the distribution is ready itself.
